gradle.properties

```diff
@@ -4,11 +4,11 @@ version=1.1.0-beta.2-SNAPSHOT
 puppycrawlCheckstyleVersion=8.18
 ballerinaGradlePluginVersion=0.9.0
 
-ballerinaLangVersion=2.0.0-beta.2-20210611-210800-644dabf5
-stdlibCacheVersion=2.1.0-beta.2-20210612-124600-5b5e819
-stdlibCryptoVersion=1.1.0-beta.2-20210612-123300-19f09bd
-stdlibLogVersion=1.1.0-beta.2-20210612-123500-04f5f9d
-stdlibRegexVersion=0.7.0-beta.2-20210612-122700-366882d
-stdlibTimeVersion=2.0.0-beta.2-20210612-122700-2c3fb15
+ballerinaLangVersion=2.0.0-beta.2-20210609-112900-2f69ff3c
```

Comment:
Any reason to downgrade the dependencies?

Reply:
Yeah this was a mistake. Fixing it now.

Reply:
Fixed with https://github.com/ballerina-platform/module-ballerina-jwt/pull/258/commits/655d7cdf980892eeba64120f833bc2f4de93891f